store/bolt/user_store_test.go

```diff
@@ -0,0 +1,112 @@
+package bolt_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/gopheracademy/congo"
+	"github.com/gopheracademy/congo/store/bolt"
+)
+
+// Ensure the store can create a user.
+func TestUserStore_CreateUser(t *testing.T) {
+	s := OpenStore()
+	defer s.Close()
+
+	u := &congo.User{
+		FirstName: "Susy",
+		LastName:  "Que",
+		Email:     "susy@que.com",
+	}
+
+	// Create user & verify ID is returned.
+	if err := s.CreateUser(u); err != nil {
+		t.Fatal(err)
+	} else if u.ID != 1 {
+		t.Fatalf("unexpected id: %d", u.ID)
+	}
+
+	// Verify user can be retrieved.
+	if u, err := s.User(1); err != nil {
+		t.Fatal(err)
+	} else if !reflect.DeepEqual(u, &congo.User{
+		ID:        1,
+		FirstName: "Susy",
+		LastName:  "Que",
+		Email:     "susy@que.com",
+	}) {
+		t.Fatalf("unexpected user: %#v", u)
+	}
```

Comment:
Then we try to look up the user by ID and verify that the data was saved correctly.